norduser/group_monitor.go

```diff
@@ -165,6 +178,8 @@ func (n *NordvpnGroupMonitor) Start() error {
 	currentGrupMembers, err := getNordvpnGroupMembers()
 	if err != nil {
 		return fmt.Errorf("getting initial group members: %w", err)
+	} else {
+		n.startForEveryGroupMember(currentGrupMembers)
 	}
```

Comment:
`else` is not needed as function wil return if condition is met
```suggestion
	}
	n.startForEveryGroupMember(currentGrupMembers)
```